src/renderer/component/categoryList/view.jsx

```diff
@@ -32,6 +35,11 @@ class CategoryList extends React.PureComponent<Props, State> {
   }
 
   componentDidMount() {
+    const { category, fetchChannel } = this.props;
+    if (category[0] === '@') {
+      fetchChannel(category);
```

Comment:
Use `makeSelectFetchingChannelClaims` to make sure we aren't already fetching for this channel.